spec/features/user_search_for_other_users_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+feature "User searches for a user", %(
+  As a user
+  I want to search for others users
+  So that I can view their profile
+
+  Acceptance Criteria
+  [ ] I must be displayed with all the users that match my search
+) do
+
+  scenario 'visitor searches for a tattoo' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.